pi-geofence/src/main/java/com/ibm/pi/geofence/LoggingConfiguration.java

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
Do you think we should move this class into the Core module, or have separate instances in each of the modules?